packages/debug/src/browser/editor/debug-breakpoint-widget.tsx

```diff
@@ -82,18 +90,17 @@ export class DebugBreakpointWidget implements Disposable {
         const inputNode = document.createElement('div');
         inputNode.classList.add('theia-debug-breakpoint-input');
         this.zone.containerNode.appendChild(inputNode);
-
-        const input = this._input = await this.createInput(inputNode);
+        this.expressionInput = this._input = await this.createInput(inputNode);
```

Comment:
This seems a bit odd. Why the two fields, `this.expressionInput` and `this._input`?

Reply:
Thanks for catching this, I had renamed the input field without seeing this member variable to const reassignment. I've reverted this change and kept the original name to avoid any breakage.